Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-gem 'omnibus', git: 'git://github.com/datadog/omnibus-ruby.git', branch: 'custom3.2.2-2'
-gem 'omnibus-software', git: 'git://github.com/datadog/omnibus-software.git', branch: 'master'
+gem 'omnibus', git: 'git://github.com/datadog/omnibus-ruby.git', branch: 'etienne/ship-licenses-and-sources'
```

Comment:
Do you mind if I do that only after merging my changes to the final branches in question ?